lunar/distribution.yaml

```diff
@@ -1255,6 +1255,22 @@ repositories:
       url: https://github.com/orocos/rtt_ros_integration.git
       version: toolchain-2.9
     status: maintained
+  srdfdom:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/srdfdom.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/srdfdom-release.git
+      version: 0.4.2-0
+    source:
+      test_pull_requests: true
```

Comment:
I have just added the buildfarm user to this org unit. Please make sure that this is the case for future PRs.